HACKING.md

```diff
@@ -33,6 +35,37 @@ Follow the instructions to install the operator, e.g.:
 $ oc apply -f /tmp/manifests/path
 ```
 
+#### Building Remotely on the Cluster
+
+To build the operator on the remote cluster, first create a buildconfig.  Use the following command, with the appropriate strings substituted for `<username>` and `<branch>`:
+
+```
+$ oc process -f hack/buildconfig.yaml \
+     -p 'GIT_URL=https://github.com/<username>/cluster-ingress-operator.git' \
+     -p 'GIT_BRANCH=<branch>' \
+      | oc -n openshift-ingress-operator create -f -
+```
```

Comment:
Could a script discover all this input using `git` so the UX can be something like:

    $ make build-cluster

Reply:
Yeah, it could.  I cannot find a good way to find the URL, but `hub browse -u <branch>` can figure it out, so I should be able to too.

Reply:
Done.